docs/en/sql-reference/20-sql-functions/10-semi-structured-functions/json-array-filter.md

```diff
@@ -0,0 +1,32 @@
+---
+title: JSON_ARRAY_FILTER
+---
+import FunctionDescription from '@site/src/components/FunctionDescription';
+
+<FunctionDescription description="Introduced or updated: v1.2.644"/>
+
+Filters elements from a JSON array based on a specified condition, returning only the elements that satisfy the condition.
+
+## Syntax
+
+```sql
+JSON_ARRAY_FILTER(<json_array>, <filter_condition>)
```

Comment:
the second param is a lambda function, it's syntax is 
```sql
param -> expr
(param1, param1 ) -> expr
```
we'd better add some explanations and relevant examples.